model/schema/highThroughputExpression.yaml

```diff
@@ -0,0 +1,578 @@
+id: https://github.com/alliance-genome/agr_persistent_schema/src/schema/highThroughputExpression.yaml
+name: High-Throughput-Expression-Dataset-Annotation
+title: High Throughput Expression Dataset Annotation
+description: Annotations for high-throughput gene expression experiments and datasets
+
+imports:
+  - core
+  - expression
+  - linkml:types
+
+prefixes:
+  alliance: 'http://alliancegenome.org/'
+  linkml: 'https://w3id.org/linkml/'
+  gff: 'https://w3id.org/gff'
+  faldo: 'http://biohackathon.org/resource/faldo#'
+  biolink: 'https://w3id.org/biolink/vocab/'
+  NLMID: 'https://www.ncbi.nlm.nih.gov/nlmcatalog/?term='
+  schema: 'http://schema.org/'
+  dct: 'http://purl.org/dc/terms/'
+  WIKIDATA_PROPERTY: 'https://www.wikidata.org/wiki/Property:'
+  obograph: 'https://github.com/biodatamodels/obograph'
+
+default_prefix: alliance
+default_range: string
+
+default_curi_maps:
+  - obo_context
+  - idot_context
+  - semweb_context
+  - monarch_context
+
+emit_prefixes:
+  - rdf
+  - rdfs
+  - xsd
+  - owl
+
+
+classes:
+
+  HighThroughputExpressionDatasetAnnotation:
+    is_a: SubmittedObject
+    description: >-
+      An entry for a published high-throughput expression dataset annotation.
+    slots:
+      - dataset_id # datasetId in agr_schemas json
+      - name # title in agr_schemas json
+      - references # publications in agr_schemas json
+      - related_note # use 'summary' note type; summary in agr_schemas json
+      - number_of_channels # numChannels in agr_schemas json
+      - sub_series # subSeries in agr_schemas json
+      - category_tags # categoryTags in agr_schemas json
+    slot_usage:
```

Comment:
dataset_id, name, and category_tags are all required fields in the agr_schema repo - should they be here too?
dateAssigned is also a required field in the old repo but I don't see an equivalent field here, is that intentional?

Reply:
@markquintontulloch Thanks for catching that. I will fix